sdk/ai/ai-inference-rest/src/modelClient.ts

```diff
@@ -19,11 +25,10 @@ import { ModelClient } from "./clientDefinitions.js";
 export default function createClient(
   endpointParam: string,
   credentials: TokenCredential | KeyCredential,
-  options: ClientOptions = {},
+  { apiVersion = "2024-05-01-preview", ...options }: ModelClientOptions = {},
```

Comment:
double checking, the api version didn't change between the two versions?

Reply:
Still 2024-05-01-preview according to [latest TypeSpec](https://github.com/Azure/azure-rest-api-specs/blob/bbdcedf51483d68a407190ce903454380f8e9ec8/specification/ai/ModelClient/main.tsp#L31)